skype/skype-ps/skype/Set-CsUser.md

```diff
@@ -407,6 +415,9 @@ Depending on your setup, those two numbers could actually be different.
 For example, in some organizations dialing 1-425-555-1297 routes your call to an Exchange Auto Attendant.
 Conversely, dialing just the extension (51297) or using Skype for Business to dial the number 1-425-555-1297 extension 51297 will route your call directly to the user.
 
+> [!NOTE]
+> Using this parameter for Microsoft Teams users will be deprecated. Please start using the new cmdlet [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment).
```

Comment:
```suggestion
**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.
```

Reply:
@JohanFreelancer9 the cmdlet Set-CsUser itself will stay, but not for Teams. So it's really the parameters that only will be deprecated